### PR TITLE
Create component scss files

### PR DESCRIPTION
**Pull request by Heathercoraje**
- [x] create components scss files for each react components 
- [x] add sass-loader in webpack.config
- [x] import style.scss as main stylesheet 
- [x] fix missing properties while converting from sass to scss

### CODE REVIEW

**Comment by veredrec** [comment]
Great job @Heathercoraje :tada: thanks!!

File: package.json [hunk 1]
```diff
@@ -3,7 +3,9 @@
   "version": "1.0.0",
   "description": "",
   "main": "index.jsx",
-  "engines" : { "node" : ">=8.4" },
+  "engines": {
```
Comment (Karyum) [inline package.json]: whats this ?
never seen it before

File: src/index.jsx [hunk 2]
```diff
@@ -3,7 +3,7 @@ import ReactDOM from 'react-dom';
 import { Provider } from 'react-redux';
 import Routes from './router.jsx';
 import Career from './containers/career.jsx';
-import Style from '../public/style.css';
+import Style from '../public/scss/style.scss';
```
Comment (Karyum) [inline src/index.jsx]: wicked !
now you can get rid of the style.css ( not scss ) file cause it's just loads of extra code

File: public/scss/_navbar.scss [hunk 3]
```diff
@@ -0,0 +1,35 @@
+$light-blue: #68829E;
+$roboto: "Roboto", sans-serif;
+
+body {
```
Comment (Karyum) [inline public/scss/_navbar.scss]: you have this also in the _career.scss file do you need it again?

File: public/scss/_welcome.scss [hunk 4]
```diff
@@ -0,0 +1,54 @@
+html, body {
```
Comment (Karyum) [inline public/scss/_welcome.scss]: also here

File: package.json [hunk 5]
```diff
@@ -42,6 +44,7 @@
     "react-icons": "^2.2.7",
     "react-redux": "^5.0.6",
     "react-router-dom": "^4.2.2",
-    "redux": "^3.7.2"
+    "redux": "^3.7.2",
+    "webpack-sass-loaders": "^1.0.0"
```
Comment (Karyum) [inline package.json]: are we sure this is the right package ?
cause in the webpack.config file you are saying it's sass-loader not wewbpack-sass-loaders

**Comment by Heathercoraje** [comment]
@Karyum .sass-cache has been removed and some duplicate codes in scss files got delelted

File: package.json (hunk 1 above)
Comment (Karyum) [reply package.json]: @Heathercoraje

File: package.json (hunk 5 above)
Comment (Karyum) [reply package.json]: it works

File: src/index.jsx (hunk 2 above)
Comment (Karyum) [reply src/index.jsx]: @Heathercoraje

File: package.json (hunk 1 above)
Comment (Heathercoraje) [reply package.json]: For Heroku

**Pull request merged**